openqa-trigger-bisect-jobs

```diff
@@ -47,28 +48,60 @@ def call(cmds, dry_run=False):
 def openqa_clone(cmds, dry_run, default_opts=['--skip-chained-deps', '--within-instance'], default_cmds=['_GROUP=0']):
     return call(['openqa-clone-job'] + default_opts + cmds + default_cmds, dry_run)
 
+def fetch_url(url, request_type='text'):
+    if url.scheme == "file":
+        decodejson = lambda c: json.loads(c)
+        path = url.geturl()
+        path = path[7:]
+        path = 'test/data/python-requests/' + path
+        with open(path, "r") as request:
+            raw = content = request.read()
```

Comment:
this looks like you want to replicate more or less what https://github.com/os-autoinst/openqa_review/blob/master/openqa_review/browser.py does with loading/saving for caching+testing purposes, is it? If we need to do the same here I wonder if there isn't already a better way in the python community to mock requests for test data